frontend/talentsearch/src/js/components/search/SearchContainer.tsx

```diff
@@ -107,15 +105,22 @@ export const SearchContainer: React.FC<SearchContainerProps> = ({
   // at the very end, in a way that confuses Cypress. Caution advised before
   // re-producing this pattern elsewhere.
   // See: https://github.com/GCTC-NTGC/gc-digital-talent/pull/4119#issuecomment-1271642887
-  const tryHandleSubmit = async () => {
+  const tryHandleSubmit = async (candidateCount: number) => {
     if (poolClassificationFilterCount === 0 || locationPreferencesCount === 0) {
       // Validate all fields, and focus on the first one that is invalid.
       searchRef.current?.triggerValidation(undefined, { shouldFocus: true });
     } else {
-      onSubmit();
+      onSubmit(candidateCount);
     }
   };
 
+  const totalCandidateCount =
```

Comment:
OK, so I like this.  :laughing:  I think technically you're supposed to also issue a couldApplicants query which will return a smaller number since some poolcandidates will correspond to the same user.  I think showing that number will be confusing and not super helpful to the user.  And it hits the database again.  :shrug:  I guess you could talk to Tristan or one of the designers about it.

Reply:
Okay thanks for catching this Peter. Should be fixed by [48bc7c7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4528/commits/48bc7c7c56653a1740819bf13371cf4241bca8e2).